README.md

```diff
@@ -83,20 +83,29 @@ you don't need to run `helm repo update`).
 
 Your pushed chart is available:
 
-    $ helm search mynewrepo 
+    $ helm search mynewrepo
     NAME                    VERSION	 DESCRIPTION
     mynewrepo/epicservice   0.7.2    A Helm chart.
 
 Note that the plugin denies push when the chart with the same version already exists
 in the repository. This behavior is intentional. It is useful, for example, in
 CI automated pushing: if someone forgets to bump chart version - the chart would
-not be overwritten. 
+not be overwritten.
 
 However, in some cases you want to replace existing chart version. To do so,
 add `--force` flag to a push command:
 
     $ helm s3 push --force ./epicservice-0.7.2.tgz mynewrepo
 
+In use cases where you share a repo across multiple AWS accounts,
+you may want the ability to define object ACLS to allow charts to persist there
+permissions across accounts.
+To do so, add `--acl="ACL_POLICY"`. The list of ACLs can be [found here](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl):
+
+    $ helm s3 push --acl="bucket-owner-full-control" ./epicservice-0.7.2.tgz mynewrepo
+
+You can also set the default ACL be setting the `S3_ACL` environment variable.
+
```

Comment:
Thanks for valuable documentation! 

As ACL applies not only to `push` command but to others too, maybe we want to move this block in a separate section in README? 

I think a good place is right before `Using alternative S3-compatible vendors` section.

Reply:
I've moved it to its own section now 👍